src/styles/elements/_header.scss

```diff
@@ -0,0 +1,82 @@
+.header {
```

Comment:
blocks (elements) file names should not start with the underscore `_`. Only additional styles (like variables, some normalize/reset, etc. should start with underscore)